Seal_of_Approval.md

```diff
@@ -0,0 +1,21 @@
+# Seal of Approval
+
+This is a list of packages in the `data.table` community.
+
+Further detail about these packages and their relationship to `data.table` can be found at [The Raft blog](https://rdatatable-community.github.io/The-Raft/#category=seal%20of%20approval).
+
+To add your package to this list, please [submit a Pull Request to The Raft](https://github.com/rdatatable-community/The-Raft/), making sure to follow the templated instructions.
+
+## Extension packages: Add to the internal functionality of `data.table`
+
+- [nc](https://github.com/tdhock/nc): Named capture regular expressions for text parsing and data reshaping.
+
+## Application packages: Use `data.table` to accomplish a particular task or analysis.
+
+## Bridge package: Translate data.table syntax to different syntax or provides helper functions for transitioning between `data.table` and another object type.
+
+- [tidyfast](https://github.com/TysonStanley/tidyfast): Fast and efficient alternatives to tidyr functions built on data.table
+
+- [dtplyr](https://github.com/tidyverse/dtplyr): A `data.table` backend for `dplyr`
+
+## Partner package: Not necessarily directly connected to `data.table`, but deliberately follows the [core philosophies of `data.table`](https://github.com/Rdatatable/data.table/blob/master/GOVERNANCE.md).
```

Comment:
This link is wrong, no?

Reply:
I believe this is referring to the six that we list in the governance.

- Time & memory efficiency
- Concise syntax (minimal redundancy in code)
- No external Imports/LinkingTo/Depends dependencies (external meaning those not maintained by the project)
- Few (if any) Suggests/Enhances dependencies
- Stable code base (strong preference for user-friendly back-compatibility with data.table itself and with old versions of R)
- Comprehensive and accessible documentation and run-time signals (errors, warnings)

Reply:
can we link that section of the file?

Reply:
I did